<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inject</class>
 <widget class="QWidget" name="Inject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>578</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>打开:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="EDIT_DLLPATH">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>26</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BTN_FILE">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>530</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="QPushButton" name="BTN_REFRESH">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>刷  新</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BTN_LOAD">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>注  入</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BTN_UNLOAD">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>卸  载</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>410</y>
     <width>291</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>账 户:   </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="EDIT_USER">
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>26</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>密 码:  </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="EDIT_PASS">
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>26</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BTN_LOGIN">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string> 自 动 登 录 </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="maximumSize">
       <size>
        <width>450</width>
        <height>380</height>
       </size>
      </property>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
